scripts/setup-adapters.sh

```diff
@@ -23,6 +23,111 @@ SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 source $SCRIPTDIR/setup-helper-functions.sh
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
 
+function install_gcs-sdk-cpp {
+  user=`whoami`
+  case "$user" in
+    root) echo 'Running as root' ; sudocmd="";;
+       *) echo 'Running as other user than root' ; sudocmd="sudo";;
+  esac
+  # install gcs dependencies
+  # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
+  # install abseil
+  echo " Installing abseil..."
+  mkdir -p ${DEPENDENCY_DIR}/abseil-cpp && cd ${DEPENDENCY_DIR}/abseil-cpp
+  curl -sSL https://github.com/abseil/abseil-cpp/archive/20211102.0.tar.gz | \
+      tar -xzf - --strip-components=1 && \
```

Comment:
For each and every of below steps it would go then:
`github_wget_and_untar 'abseil/abseil-cpp/archive/20211102.0.tar.gz' 'abseil-cpp'`